src/commands/platforms/destroy.ts

```diff
@@ -8,7 +8,10 @@ import BaseCommand from '../../base-command';
 export default class PlatformDestroy extends BaseCommand {
   static aliases = ['platforms:deregister', 'platform:destroy', 'platforms:destroy'];
   static description = 'Deregister a platform from Architect';
-
+  static examples = [
+    'architect platform:destroy --account=myAccount architect',
+    'architect platforms:deregister --account=myAccount --type=kubernetes --auto-approve --force architect',
```

Comment:
There does not seem to be a flag `--type=kubernetes` for this command.

Reply:
removed --type from platforms:deregister examples